python/ray/tune/integration/xgboost.py

```diff
@@ -54,9 +68,15 @@ def __init__(self,
             metrics = [metrics]
         self._metrics = metrics
 
-    def _get_report_dict(self, env):
-        # Only one worker should report to Tune
-        result_dict = dict(env.evaluation_result_list)
+    def _get_report_dict(self, evals_log):
+        if isinstance(evals_log, OrderedDict):
+            # xgboost>=1.3
+            result_dict = flatten_dict(evals_log, delimiter="-")
+            for k in list(result_dict):
+                result_dict[k] = result_dict[k][0]
```

Comment:
any reason why we take the first value?

Reply:
In my tests there was only one. But right, maybe we should take the last one